.github/ISSUE_TEMPLATE/2-playback-issue.yml

```diff
@@ -0,0 +1,104 @@
+name: Playback Issue
+description: You have playback issues with some files
+labels:
+  - playback
+
+body:
+  - type: textarea
+    id: playbug
+    validations:
+      required: true
+    attributes:
+      label: Describe The Bug
+      description: A clear and concise description of what the bug is.
+
+  - type: textarea
+    id: media
+    validations:
+      required: true
+    attributes:
+      label: Media Information
+      description: Please paste any ffprobe or MediaInfo logs.
+      render: bash
```

Comment:
```suggestion
      description: Please include any ffprobe or MediaInfo logs.
```

Reply:
Afaik, when adding the `render` option, GitHub will no longer allow a File Upload as a submission in that Text Field.
In the end its a matter of choice, if the team rather wants a bash Codeblock, or a bare Textfield where Users Potentialy Upload logfiles to.

In my personal Oppinion, rendering to bash might the better choice.

Reply:
The third option is to add a `upload log file(s) here`